backend/DB/migrations/20201202124324_message.js

```diff
@@ -0,0 +1,15 @@
+
+exports.up = async function(knex) {
+    let hasTable = await knex.schema.hasTable('message')
+    return !hasTable ? knex.schema.createTable('message', (table)=>{
+        table.integer('student_id').unsigned().references('id').inTable('student').notNullable();
```

Comment:
We will have problems here with the seeds. We need to decide what is our delete strategy in this case, meaning if a student or an employer get deleted, what happens to their messages and conversations? they get deleted as well? I would suggest yes, and in that case, we should add on the 2 foreign keys "onDelete('CASCADE'). What do you think?

Reply:
I added this